index.js

```diff
@@ -60,7 +60,11 @@ function toMajor (mcVersion, preNetty, typeArg) {
   } else if (preNetty && preNettyVersionsByProtocolVersion[type][version]) {
     return toMajor(preNettyVersionsByProtocolVersion[type][version][0].minecraftVersion, preNetty, type)
   } else if (!preNetty && postNettyVersionsByProtocolVersion[type][version]) {
-    return toMajor(postNettyVersionsByProtocolVersion[type][version][0].minecraftVersion, preNetty, type)
+    if (version === 5) { // Prtocol version 5 is ambiguous between 1.7 and early 1.8 snapshots, assume 1.7
+      majorVersion = '1.7'
```

Comment:
What about taking the last non snapshot version ?

Reply:
This does that, the result of setting majorVersion to 1.7 here is:

```js
const mcData = require('minecraft-data')
console.log(mcData(5).version)
```

```
{ version: 5, minecraftVersion: '1.7.10', majorVersion: '1.7' }
```